### ISSUE TITLE
Add Markers in Realtime

### ISSUE THREAD

**Issue by halimpuckjava** [sothawo/mapjfx#55]
hi

how i can add marker in realtime (like stream-event from server and for each event i add maker in mapview)

**Comment by sothawo** [comment]
can you please provide more information?
which version of mapjfx with which Java version?
can you provide a minimal sample code to reproduce this error?

**Comment by halimpuckjava** [comment]
i use version 2.7.1 

i have server send events to client, so i want client display marker when event receive in realtime

**Comment by sothawo** [comment]
ok, checking the code
```
initialized.set(true);
setMapTypeInMap();
setCenterInMap();
setZoomInMap();
```
I see that the initialized property is set to `true`before the center and zoom are set it should be done after that; have to rework that

**Comment by halimpuckjava** [comment]
initialized id readonly so u can't modify

**Comment by sothawo** [comment]
fixed with #56
